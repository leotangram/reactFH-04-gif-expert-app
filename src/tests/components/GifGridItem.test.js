import React from 'react'
import { shallow } from 'enzyme'
import GifGridItem from '../../components/GifGridItem'

describe('GifGridItem Component', () => {
  test('should show the component correctly', () => {
    const title = 'Title'
    const url = 'https://asdasd.asd.asd'
    const wrapper = shallow(<GifGridItem title={title} url={url} />)
    expect(wrapper).toMatchSnapshot()
  })
})
