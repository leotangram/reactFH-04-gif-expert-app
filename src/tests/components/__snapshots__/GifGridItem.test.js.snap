// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifGridItem Component should show the component correctly 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="Title"
    src="https://asdasd.asd.asd"
  />
  <p>
    Title
  </p>
</div>
`;
